packages/frontend/src/app/build/tests-list/test-details/test-details.component.html

```diff
@@ -0,0 +1,73 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!-- CLOSE BUTTON -->
+<div id="close-btn-container">
+    <mat-icon id="close-btn" [mat-dialog-close]="true">close</mat-icon>
+</div>
+
+<!-- NAME -->
+<div class=".outer-field">
+    <span
+        id="name"
+        [class.failing]="test.failing"
+        [class.flaky]="!test.failing && test.flaky"
+        [class.success]="!test.failing && !test.flaky">
+        {{test.name}}
+    </span>
+</div>
+
+<mat-divider></mat-divider>
+
+<!-- FAILURE DETAILS -->
+<div *ngIf="test.failing" class=".outer-field">
+    <section>
+        <!-- HEADER -->
+        <header id="error-header">Error message</header>
+
+        <!-- MESSAGE -->
+        <textarea readonly id="error-msg-textarea" rows="5" >Error with ...&#013;&#010;Unable to ...
```

Comment:
How does this interaction work now, is it a pop up like in the screen shot?

I'm happy with this as a starting point, but to @JustinBeckwith's point, I can imagine we make an effort to avoid pop ups eventually 😄

Reply:
Yes, it is.
We will use an expansion eventually instead.